### DISCUSSION TITLE
Extend .server folder naming convention

### DISCUSSION THREAD

**Discussion by thefrana** [remix-run/remix#9370] [Proposals]
Hello,

I'd like to propose an extension to the current `.server` naming convention. Currently, folders must be named strictly as `.server`. Would it be feasible to also permit naming such as `utils.server`? This approach might offer more convenience compared to the existing `.server/utils` folder structure. I would appreciate your thoughts on this suggestion.

Thank you!

**Comment by kiliman** [comment]
It already supports that. The `.server` is a _suffix_ so you can name it `utils.server`. 

You can have a file named `utils.server.ts` or a folder named `utils.server/index.ts`

**Reply by kiliman** [reply]
BTW: the `.server` folder suffix only applies to Vite. Only filenames are supported in the _Classic_ compiler.

**Reply by thefrana** [reply]
I do think that's right. I just tried to create some constant under `app/utils.server/test.ts` and then import the constant in a route. It works. However, when I change the folder name to just `.server`, it throws compilation error as expected.

**Reply by kiliman** [reply]
Did you check the client bundle to see if your test constant was there?
